snap/info.go

```diff
@@ -231,6 +232,18 @@ type DownloadInfo struct {
 	Sha3_384 string `json:"sha3-384,omitempty"`
 }
 
+// DeltaInfo contains the information to download a delta
+// from one revision to another.
+type DeltaInfo struct {
+	FromRevision    int    `json:"from-revision,omitempty"`
+	ToRevision      int    `json:"to-revision,omitempty"`
```

Comment:
I've made the other changes, but not (yet) this one. Initially the store will be providing just one delta iff one exists from the current client revision to the latest (eg. 24->26). Once we're able to test that, we'll update to include multiple consecutive deltas (as per the test, 24->25, 25->26) if there is not a single delta which matches. In both of those cases, here in snapd client we *could* just store the ToRevision (assume the first is from our current revision and apply in order) or the FromRevision (apply in order and assume the last gets us to the correct revision), but I'd personally rather be explicit. If you still think it's simpler to just have one, I'll update it. I'll check back in tonight to try to chat about it.